### PR TITLE
EBL-281: stack blocker does not work

### PR DESCRIPTION
**Pull request by maorshap**

### CODE REVIEW

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java [hunk 1]
```diff
@@ -125,16 +125,22 @@ private boolean isBlockerInScope(Blocker blocker, Deployment deployment) {
...
                     if (blockerInjectableCommons.getStackService().getEnvironmentsStack(blocker.getStackId()).getEnvironments().stream().anyMatch(environmentId -> environmentId == deployment.getEnvironmentId())) {
                         environmentToCheck = deployment.getEnvironmentId();
+                    } else {
+                        return false;
                     }
                     break;
                 case SERVICES:
-                    if (blockerInjectableCommons.getStackService().getServicesStack(blocker.getStackId()).getServices().stream().anyMatch(serviceId -> serviceId == deployment.getServiceId())) {
+                    List<Integer> servicesIdList = blockerInjectableCommons.getStackService().getServicesStack(blocker.getStackId()).getServices();
+                    if (servicesIdList.stream().anyMatch(serviceId -> serviceId == deployment.getServiceId())) {
```
[the remark this answers is not in the document]
Comment (maorshap) [reply apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:140]: Fixed.

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java [hunk 2]
```diff
@@ -172,6 +178,7 @@ private boolean isBlockerInScope(Blocker blocker, Deployment deployment) {
             return true;
         }
 
+        ///////HERE!!!!/////
```
[the remark this answers is not in the document]
Comment (maorshap) [reply apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:181]: fixed.

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java [hunk 3]
```diff
@@ -148,9 +157,7 @@ private boolean isBlockerInScope(Blocker blocker, Deployment deployment) {
             return false;
         }
 
-        if (isUserAllowedToOverride(deployment, blocker)) {
-            return false;
-        }
+
```
Comment (mayhuberman) [inline apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:160]: unnecessary new line

File: ui/app/scripts/config.js [hunk 4]
```diff
@@ -1,9 +1,6 @@
...
     var CONFIG = {
-        appUrl: document.location.protocol + "//" + document.location.host + "/api/",
-//        appUrl: 'http://localhost:8081/',
-        wsUrl: document.location.protocol == "https:" ? "wss:"  + "//" + document.location.host + "/ws/" : "ws:" + "//" + document.location.host + "/ws/",
-//        wsUrl: 'ws://localhost:8082/',
-//        hawtioUrl: 'http://localhost:8080/'
-        hawtioUrl: document.location.protocol + "//" + document.location.host + "/hawtio/"
+        appUrl: 'http://localhost:8081/',
+        wsUrl: 'ws://localhost:8082/',
+        hawtioUrl: 'http://localhost:8080/'
```
Comment (mayhuberman) [inline ui/app/scripts/config.js:5]: you accidentally changed the config file

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java [hunk 5]
```diff
@@ -125,16 +129,21 @@ private boolean isBlockerInScope(Blocker blocker, Deployment deployment) {
             serviceToCheck = blocker.getServiceId();
         }
 
+        logger.info("ShapLog: Blocker name {}", blocker.getName());
```
Comment (mayhuberman) [inline apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:132]: remove please :)

File: ui/app/scripts/config.js (hunk 4 above)
Comment (maorshap) [reply ui/app/scripts/config.js:5]: I do not see this config file in my project.

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java (hunk 3 above)
Comment (maorshap) [reply apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:160]: fixed.

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java (hunk 5 above)
Comment (maorshap) [reply apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:132]: fixed.

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java [hunk 6]
```diff
@@ -114,6 +114,11 @@ private boolean isUserAllowedToOverride(Deployment deployment, Blocker blocker)
 
     @SuppressWarnings("RedundantIfStatement")
     private boolean isBlockerInScope(Blocker blocker, Deployment deployment) {
+        if (isUserAllowedToOverride(deployment, blocker)) {
+            return false;
+        }
+
+
```
Comment (mayhuberman) [inline apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java:121]: unnecessary new line

File: ui/app/scripts/config.js (hunk 4 above)
Comment (mayhuberman) [reply ui/app/scripts/config.js]: ui/app/scripts/config.js

Comment (maorshap) [reply ui/app/scripts/config.js:6]: pushed

File: apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java (hunk 6 above)
Comment (maorshap) [reply apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java]: fixed.

**Pull request merged**